File: app/src/main/java/com/example/eccard/filmesfamosos/data/network/model/MovieTrailersReviewResponse.java
package com.example.eccard.filmesfamosos.data.network.model;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MovieTrailersReviewResponse {


    @SerializedName("results")
    @Expose
    private List<TrailerResult> results = null;

    public List<TrailerResult> getResults() {
        return results;
    }

    public void setResults(List<TrailerResult> results) {
        this.results = results;
    }
}

File: app/src/main/java/com/eccard/popularmovies/data/network/model/network/MovieTrailersReviewResponse.kt
package com.eccard.popularmovies.data.network.model.network

import com.eccard.popularmovies.data.network.model.TrailerResult

data class MovieTrailersReviewResponse(
        val id: Int,
        val page: Int,
        val results: List<TrailerResult>,
        val total_pages: Int,
        val total_results: Int
)